metrics/grafana/README.md

```diff
@@ -0,0 +1,11 @@
+# TiKV Grafana Dashboard
+
+The "TiKV Details" dashboard is generated by the `tikv_details.dashboard.py`
+Python script.
+
+## Updating the Dashboard
+
+To add or update panels on the dashboard, make your changes in
+`tikv_details.dashboard.py` and then run `./scripts/gen-tikv-details-dashboard`.
+
+Please avoid manually modifying `tikv_details.json`.
```

Comment:
How about adding a SHA1 checksum check in `make format` or else to make sure the file is not changed

Reply:
Maybe we should make the check of tikv_details.json as part of the make script (in all debug|release|dist_release)
Otherwise someone may change the json file directly without changing the .py file.

Reply:
Appreciate your feedback, addressed in latest commits.